colour/algebra/common.py

```diff
@@ -74,16 +74,18 @@
 """
 
 
-def get_sdiv_mode() -> Literal[
-    "Numpy",
-    "Ignore",
-    "Warning",
-    "Raise",
-    "Ignore Zero Conversion",
-    "Warning Zero Conversion",
-    "Ignore Limit Conversion",
-    "Warning Limit Conversion",
-]:
+def get_sdiv_mode() -> (
+    Literal[
+        "Numpy",
+        "Ignore",
+        "Warning",
+        "Raise",
+        "Ignore Zero Conversion",
+        "Warning Zero Conversion",
+        "Ignore Limit Conversion",
+        "Warning Limit Conversion",
+    ]
+):
```

Comment:
**Minor**

This is probably incorrectly formatted!?

Reply:
I don't see this change anymore, but I did clean this up with a type alias